src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Pagination.css';
+
+export const Pagination = (props) => {
+  const { total, page, perPage, onPageChange, withInfo } = props;
+
+  const handleButtonClick = pageNumber => onPageChange(pageNumber);
+
+  const renderButtons = () => (
+    new Array(Math.ceil(total / perPage))
+      .fill('')
+      .map((it, index) => {
+        const buttonClassName = (index + 1 === page)
+          ? 'ui red basic button'
+          : 'ui green basic button';
+
+        const key = index + 1;
+
+        return (
+          <button
+            type="button"
+            key={key}
+            className={buttonClassName}
+            onClick={() => handleButtonClick(index + 1)}
+          >
+            {index + 1}
+          </button>
+        );
+      })
+  );
+
+  const renderInfo = () => {
+    if (!withInfo) {
+      return null;
+    }
+
+    const min = page * perPage - perPage + 1;
+    const max = Math.min(page * perPage, total);
+
+    return `${min} - ${max} of ${total}`;
+  };
+
+  const handlePrevButtonClick = () => onPageChange(page - 1);
+
+  const handleNextButtonClick = () => onPageChange(page + 1);
```

Comment:
there is no need to create those methods

Reply:
done